utils/sl/components/__init__.py

```diff
@@ -24,6 +24,7 @@
 import sys
 import os
 import platform
+import distro
```

Comment:
(pylint-import-error) Unable to import 'distro'

Reply:
No package python3-distro available.